src/main/flight/mixer.c

```diff
@@ -435,7 +437,21 @@ static void applyMixerAdjustmentLinear(float *motorMix, const bool airmodeEnable
                 motorMix[i] = scaleRangef(throttle, 0.0f, 1.0f, motorMix[i] + ABS(motorMix[i]), motorMix[i] - ABS(motorMix[i]));
             }
         }
-        motorMix[i] /= motorMixNormalizationFactor;
+        if (airmodeEnabled || throttle > 0.5f) {
+            if motorMix[i] > maxMotor {
```

Comment:
Parenthesis needed

Reply:
oh man, I'm getting to used to rust, lol, I'll fix it.